images/bootstrap/OWNERS

```diff
@@ -1,11 +1,11 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 reviewers:
-- amwat
 - spiffxp
 - BenTheElder
 - MushuEE
 approvers:
-- amwat
 - spiffxp
-- BenTheElder
\ No newline at end of file
+- BenTheElder
+emeritus_approvers:
+- amwat
```

Comment:
empty line?

Reply:
🤷‍♂️  the file didn't have one before.

Reply:
I normalized these files to have one anyhow